plugins/ccc35.py

```diff
@@ -0,0 +1,87 @@
+from .helpers import public_api
+from ircbot.persist import Persistent
+from ircbot.plugin import BotPlugin
+from datetime import datetime, timedelta
+import pytz
+import dateutil
+
+
+class CCC35(BotPlugin):
+    end_url = r'(?:$|\s|\)|\]|\})'
+    TZ = pytz.timezone('Europe/Berlin')
+    DAY0 = datetime(2018, 12, 26, tzinfo=TZ)
+    FAHRPLAN_CACHE = None
+
+    def format_talk(self, talk):
+        talk['start_time_dt'] = dateutil.parser.parse(talk['start_time'])
+        talk['end_time_dt'] = dateutil.parser.parse(talk['end_time'])
+        talk['relative_day'] = self.get_day(talk['start_time_dt'])
+        fmt = "«{title}» [Day {relative_day}, {start_time_dt:%H:%M}] in {room[name]}"
+        return fmt.format(**talk)
+
+    def get_day(self, dt=None):
+        if dt is None:
+            dt = datetime.now(self.TZ)
+        return (dt - self.DAY0).days
+
+    def get_fahrplan(self, force=False):
+            with Persistent('fahrplan.json') as fahrplan:
+                if not fahrplan or force:
+                    x = yield from public_api("https://fahrplan.events.ccc.de/congress/2018/Fahrplan/events.json")
+                    x['update_time'] = datetime.now(self.TZ).isoformat()
+                    fahrplan.update(x)
+            return fahrplan
+
+    @BotPlugin.command(r'.*https?://fahrplan\.events\.ccc\.de/congress/2018/Fahrplan/events/(\d+).html' + end_url)
+    def ccc_fahrplan(self, msg):
+        fahrplan = yield from self.get_fahrplan()
+        events = fahrplan['conference_events']['events']
+        event_id = int(msg.args[0])
+        for event in events:
+            if event['id'] == event_id:
+                msg.reply(self.format_talk(event))
+                break
+
+    @BotPlugin.command(r'\!day$')
```

Comment:
Usually LechBot commands are in french

Reply:
J'ai gardé "day" parce que même en francais à l'oral on dit "day" mais j'ai traduit le reste :)